pyproject.toml

```diff
@@ -4,6 +4,7 @@ build-backend = "scikit_build_core.build"
 
 [project]
 name = "cmake"
+version = "3.29.0"
```

Comment:
When we need to created patch release like `3.29.0.1`[^1], I am assuming we will update this entry accordingly ?

[^1]: https://github.com/scikit-build/cmake-python-distributions/tree/3.29.0.1

Reply:
Yes